specs/updated_cluster_list_implementation_with_upadated_API_response

```diff
@@ -0,0 +1,141 @@
+// vim: tw=79
+:imagesdir: ./images/
+
+= Tendrl Definitions: Display list of unmanaged gluster clusters discovered by Tendrl
+
+Display list of unmanaged gluster clusters present in Tendrl. The list should have functionality for 
+
+* filtering by name.
+* sorting by name, status, cluster version and managed.
+
+Cluster list will be sorted by name by default.
+
+View design will look like:
+
+image::upadtedClusterList.png[Upadted Cluster List]
+
+== Problem description
+
+Tendrl should show list of unmanaged gluster clusters present in Tendrl along with their details.
+
+The details shown for each cluster in UI is -
+
+* Cluster Name
+* Status
+* Managed State
+* Cluster Version
+* Import button
+* Message to state cluster configurable state for eg. ready to import, cluster misconfigured.
+
+The workflow is as follows:
+
+* API will provide the list of clusters to the Tendrl.
+
+* User can click on expand icon of each cluster record to see the events/notifications and hosts associated with it.
+
+* In the hosts tab, host name and IP address will be displayed.
+
+* If Managed state is "No" and cluster is ready to import, Import button will be displayed.
+
+* If Managed state is "No" and cluster is not properly configured, Import button will be disabled.
+
+* Clicking Import button will take to import cluster workflow.
+
+== Use Casess
+
+Users can see the list of unmanaged gluster cluster present in Tendrl. They can also 
+
+* filter by name.
+* sort by name, status, cluster version and managed.
+
+== Proposed change
+
+* Will use the existing cluster list view which will now display the unmanaged clusters as well.
+
+Tendrl API related changes:
+
+* Get API call for getting cluster list.
+
+Tracked in github issue: https://github.com/Tendrl/api/issues/234
+
+=== Alternatives
+
+None.
+
+=== Data model impact:
+See 'Impacted Modules' section where all the details are populated.
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+/api/1.0/GetClusterList - fetch the list of clusters
+
+Sample API Response - required.
+
+API support present for -
+
+* Status
+* Cluster Name
+* Cluster Version
+
+Required from API - 
+
+* Managed State
+* Message to state cluster configurable for eg. ready to import, cluster misconfigured.
+* Host IP address for displaying it under "Hosts" tab of expanded view.
```

Comment:
I feel ideally we should show FQDN here (or the IP to which this fqdn resolves to in case IP only to display)